src/agent.js

```diff
@@ -0,0 +1,9 @@
+// agent.js
+import https from 'https';
+import config from './config.js';
+
+const agent = new https.Agent({
+  rejectUnauthorized: config.NODE_ENV === 'production',
```

Comment:
Why do you want to only do this in production?

Reply:
because i was behind a proxy